presto-hive/src/main/java/com/facebook/presto/hive/HiveTableHandle.java

```diff
@@ -27,14 +28,27 @@
 {
     private final String schemaName;
     private final String tableName;
+    private final Optional<HiveAnalyzePropertiesHandle> analyzeHandle;
```

Comment:
Calling this inner object a "handle" is a bit confusing since it's not a handle, it's just a bean. It would probably be cleaner to inline it here as `analyzePartitions`.

Reply:
I made it a separate class because I though we might add new analyze properties in the future and it would be good if we can wrap all of them in a class. I can inline `analyzePartitions` since we don't have the need now.

Reply:
It's fine to have a class if you or anyone else has specific ideas of what we would add. In that case, let's name it `HiveAnalyzeProperties` and `analyzeProperties`, as "handle" is confusing. Otherwise, inline it, since that makes it easier to read and we avoid adding stuff speculatively as it often goes unused and just clutters the code.